README.md

```diff
@@ -48,17 +48,13 @@ package.
 
 To run the simulation with ros one can launch:
 ```bash
-hppcorbaserver
-gepetto-gui
```

Comment:
You don't need hpp anymore?

Reply:
the launch file you wrote for the mpc node is already launching hppcorbaserver and gepetto-gui

Reply:
ok but then look at the final instruction:
`ROS_NAMESPACE=/ctrl_mpc_linearized rosrun agimus_controller hpp_agimus_controller_node`
It's not using `roslaunch` but `rosrun`.
Hence if you strictly follow these lines the `hppcorbaserver` and the `gepettoi-gui` won't be started, are they?

Reply:
they are started

Reply:
ok my bad, they aren't, I probably had it launched in background